server/api/config.go

```diff
@@ -46,7 +46,10 @@ func newConfHandler(svr *server.Server, rd *render.Render) *confHandler {
 }
 
 func (h *confHandler) Get(w http.ResponseWriter, r *http.Request) {
-	h.rd.JSON(w, http.StatusOK, h.svr.GetConfig())
+	config := h.svr.GetConfig()
+	// Schedulers field for reducing output
+	config.Schedule.Schedulers = nil
```

Comment:
Could you add a query params to show the hide information? In some situations maybe we want to know it.

Reply:
To make the API compatible, only pd-ctl config show command will hide the schedulers-v2 field. `pd-ctl config show all` will output all information.